### PR TITLE
Improve structure a1

### LINKED ISSUE #3: Update structure
**Describe the feature or change you'd like**
Improve the component structure and add basic components.

### PR DESCRIPTION
**Pull request by Aichnerc**
**Checklist:**

 
 - [x] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [ ] Have you updated or added documentation for the change?
 - [x] Have you tested your changes with successful results?

**Type of Changes:**

 
 - [x] Bug fix (non-breaking change which fixes an issue)
 - [x] New feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change)

**What is the current behavior? (link to any open issues here)**
- Currently there is no cookie modal.
- Currently there is no Google Analytics and Facebook Pixel implementation.
- Currently our ScrollToTop fix is not implemented.
- Currently the application crashes because of a variable typo.
- Currently the project uses MDB React Free.

**What is the new behavior (if this is a feature change)?**
- A cookie modal has been added.
- The basic Google Analytics and Facebook Pixel functionality has been implemented.
- The ScrollToTop fix has been implemented.
- The variable typo has been resolved.
- The project now uses MDB React Pro and MDB React Sortable.

**Other information**:
- The structure has been tested and approved. #3 
- The version has been updated to v0.1.0 - we start from here. This is a pre-release.

### CODE REVIEW

**Review by kleberbaum — APPROVED** [review]
Should be ok to merge.

**Review by Aichnerc — COMMENTED** [review]
LGTM

File: src/App.js [hunk 1]
```diff
@@ -5,28 +5,92 @@ import React from "react";
...
+// Facebook Pixel
+import ReactPixel from "react-facebook-pixel";
+
 //> Components
-/**
- * Footer: Global Footer
- * Navbar: Global navigation bar
- */
-import { Footer, Navbar } from "./components/molecules";
+// Navbar, Footer and Cookie Modal
```
Comment (pinterid) [inline src/App.js]: Write "Modal" in lowercase.

File: src/App.js [hunk 2]
```diff
@@ -5,28 +5,92 @@ import React from "react";
...
+
+      userId = sha256.create();
+      localStorage.setItem("userId", userId);
+    }
+
+    // Check cookies
+    let cookie = localStorage.getItem("cookie");
+
+    if (cookie) {
+      cookie = JSON.parse(cookie);
```
Comment (pinterid) [inline src/App.js:44]: Missing newline.

File: src/components/molecules/CookieModal/index.jsx [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+} from "mdbreact";
+
+//> CSS
+import "./cookie.scss";
+//#endregion
+
+//#region > Components
+class ModalPage extends React.Component {
+  state = {
+    modal1: localStorage.getItem("cookie") ? false : true,
```
Comment (pinterid) [inline src/components/molecules/CookieModal/index.jsx]: Why modal1? Is there any other modal in this file?

**Review by pinterid — APPROVED** [review]
Looks good.

**Review by schettn — APPROVED** [review]
💮

**Pull request merged**